packages/driver/cypress/plugins/server.js

```diff
@@ -323,10 +344,12 @@ httpPorts.forEach((port) => {
   })
 })
 
-const httpsApp = createApp(httpsPort)
-const httpsServer = httpsProxy.httpsServer(httpsApp)
+httpsPorts.forEach((port) => {
+  const httpsApp = createApp(port)
```

Comment:
Do we need to create a full new app for these tests? Could we dumb this down create an express app with the two/three routes needed for this test?

Reply:
The issue is we need a different port on `https`, which we currently do not have to verify same-site cookie behavior for differing ports. The added routes in the server aren't the reason for the new app, which I don't think is obvious at all. I'm thinking some documentation/comments are needed here.

Reply:
added some docs in ec8b751c32